src/app/components/reports/pages/event-summary/EventSummaryReportParameters.ts

```diff
@@ -0,0 +1,174 @@
+import { HttpParams } from "@angular/common/http";
+import { Params } from "@angular/router";
+import { Filters, InnerFilter } from "@baw-api/baw-api.service";
+import { toBase64Url } from "@helpers/encoding/encoding";
+import { filterDate, filterTime } from "@helpers/filters/audioRecordingFilters";
+import { filterAnd, filterModelIds } from "@helpers/filters/filters";
+import { isInstantiated } from "@helpers/isInstantiated/isInstantiated";
+import { Id } from "@interfaces/apiInterfaces";
+import { EventSummaryReport } from "@models/EventSummaryReport";
+import { DateTime, Duration } from "luxon";
+
+export enum ChartType {
```

Comment:
This comment was originally because we were not going to emit visible charts in the QSP's

However, since we are emitting visible charts in the QSP's, I'd like to retain this an an enum.

While chart QSP's will technically be a string, I'd like to have either an enum or type to represent that valid chart QSP strings are not free-form and have a limited domain